help/sources/tutorials/ui/create/analytics/mixpanel.md

```diff
@@ -0,0 +1,121 @@
+---
+keywords: Experience Platform;home;popular topics;sources;connectors;source connectors;sources sdk;sdk;SDK
+title: Self-service documentation template for the UI
+description: Learn how to create a Mixpanel source connection using the Adobe Experience Platform UI.
+hide: true
+hidefromtoc: true
+---
+# Create a *Mixpanel* source connection in the UI
+
+This tutorial provides steps for creating a *Mixpanel* source connector using the Platform user interface.
+
+## Overview
+
+[Mixpanel](https://www.mixpanel.com) is a product analytics tool that enables capturing data on how users interact with a digital product. Mixpanel allows you to analyze this product data with simple, interactive reports that let you query and visualize the data with just a few clicks.
+
+This Adobe Experience Platform [sources](https://experienceleague.adobe.com/docs/experience-platform/sources/home.html?lang=en) leverages the [Event Export API > Download](https://developer.mixpanel.com/reference/raw-event-export) to download your event data as it is received and stored within Mixpanel, along with all event properties (including distinct_id) and the exact timestamp the event was fired into Experience Platform.
+
+Mixpanel uses bearer tokens as an authentication mechanism to communicate with the Mixpanel Event Export API.
+
+## Prerequisites
+
+Before you start configuring the extension you need to have a Mixpanel account. If you do not have one already, go to the Mixpanel [register](https://mixpanel.com/register/) page to register and create your Mixpanel account.
+
+### Gather required credentials
+
+In order to connect *Mixpanel* to Platform, you must provide values for the following connection properties:
+
+| Credential | Description | Example |
+| --- | --- | --- |
+| Host | Mixpanel Raw Data Export API endpoint. <br/><br/>Refer to the [Mixpanel API reference](https://developer.mixpanel.com/reference/overview) page, then scroll to the *Raw Data Export API* section if you require any guidance. | *https://data.mixpanel.com*|
+| Username | Service account Username. <br/><br/>Refer to the [Service Accounts](https://developer.mixpanel.com/reference/service-accounts#authenticating-with-a-service-account) page if you require any guidance. | *`xxxxxxxxxxx`.6d4ee7.mp-service-account* |
```

Comment:
Any reason why only `xxxxxxxxxxx` is backticks? If it's means to hide sensitive values, we could maybe just randomize some characters instead of using `xxxxxxxxxxx`